lib/mongoose.js

```diff
@@ -1167,6 +1167,19 @@ Mongoose.prototype.Mixed = SchemaTypes.Mixed;
 
 Mongoose.prototype.Date = SchemaTypes.Date;
 
+/**
+ * The Mongoose Double [SchemaType](https://mongoosejs.com/docs/schematypes.html). Used for
+ * declaring paths in your schema that should be 64-bit IEEE 754-2008 floating points.
+ *
+ * #### Example:
+ *
+ *     const vehicleSchema = new Car({ gasLevel: mongoose.Double });
+ *
+ * @property Double
+ * @api public
+ */
+Mongoose.prototype.Double = SchemaTypes.Double;
```

Comment:
Same comments as on the int32 PR - we shouldn't be exporting Double as a top-level key on the Mongoose object

Reply:
Sounds good!